src/buffers/TranscodedBuffer.h

```diff
@@ -20,8 +20,7 @@ namespace timeshift {
   public:
     TranscodedBuffer() : DummyBuffer()
     {
-      m_profile << "&profile=" << g_iResolution << "p";
-      XBMC->Log(LOG_INFO, "TranscodedBuffer created! %s", m_profile.str().c_str());
+      XBMC->Log(LOG_NOTICE, "TranscodedBuffer created");
```

Comment:
LOG_NOTICE should no longer be used in PVR addons in matrix

Reply:
Fixed in a commit after 4.3.7